lib/tasks/solid_errors_tasks.rake

```diff
@@ -0,0 +1,18 @@
+namespace :solid_errors do
+  desc "Copy and run migrations from Solid Errors"
+  task install_migrations: :environment do
+    source = File.expand_path("../../solid_errors/db/migrate", __FILE__)
+    destination = File.join(Rails.root, "db", "migrate")
+
+    Dir.glob("#{source}/*.rb").each do |migration|
+      filename = File.basename(migration)
+      unless File.exist?(File.join(destination, filename))
+        puts "Copying #{filename} from Solid Errors to Rails application"
+        FileUtils.cp(migration, destination)
+      end
+    end
+
+    puts "Running migrations from Solid Errors"
+    Rake::Task["db:migrate"].invoke
```

Comment:
This makes me nervous, as it would run any and all pending migrations in the host applications, not just the Solid Errors migrations. I think something either like this:
```shell
rake db:migrate:up VERSION=20090408054532
```
— from: https://stackoverflow.com/a/754004/2884386

or like this:
```ruby
require "#{Rails.root.to_s}/db/migrate/XXXXX_my_migration.rb"
AddFoo.migrate(:up)
```
— from: https://stackoverflow.com/a/58004549/2884386

And putting this inside of the iteration loop.